internal/testplanet/gen_identities.go

```diff
@@ -33,9 +34,48 @@ func main() {
 
 		package testplanet
 		
-		var pregeneratedIdentities = NewIdentities(
+		var (
```

Comment:
maybe use `var` near the variable name, it'll be probably easier to read the generating part that way

Reply:
:+1: